vespaclient-container-plugin/src/main/java/com/yahoo/vespa/http/server/ErrorCode.java

```diff
@@ -30,4 +35,16 @@ public boolean isTransient() {
         return _transient;
     }
 
+    static ErrorCode fromBusError(Error mbusError) {
+        Collection<Integer> fatalsHandledAsTransient = Set.of(
+                com.yahoo.messagebus.ErrorCode.SEND_QUEUE_CLOSED,
+                com.yahoo.messagebus.ErrorCode.ILLEGAL_ROUTE,
+                com.yahoo.messagebus.ErrorCode.NO_SERVICES_FOR_ROUTE,
+                com.yahoo.messagebus.ErrorCode.NETWORK_ERROR,
+                com.yahoo.messagebus.ErrorCode.SEQUENCE_ERROR,
+                com.yahoo.messagebus.ErrorCode.NETWORK_SHUTDOWN,
+                com.yahoo.messagebus.ErrorCode.TIMEOUT);
```

Comment:
Can we risk that a client expects timeouts to eventually be fatal to avoid resending "forever" if the target cluster is having issues?